### DISCUSSION TITLE
Duplicate identifiers

### DISCUSSION THREAD

**Discussion by Pluegpu** [jef/streetmerchant#2511] [Help]
Ok, so I sorted out why one store was not visible kind of. 

I changed the identifier to an existing store and tried to run the script, it failed bacause it had duplicate identifiers of course.

I change the identifier on the existing store, also changed the identifier in the indexfile.

That left me with a new store, old identifier and I then tried to run the script. Worked like a charm.

That means that the identifiers are also declared in a file somewere when adding a new store. Does anyone know which file?

**Comment by Aragorn450** [comment]
The identifiers are only defined in the store file. However, you have to also include the file in `src/store/model/index.ts` and then add it to the `storeList` that's also defined in that file.

Follow how one of the others do it. For instance, `acompc` or `zotac`. As long as you create the store file like the others in `src/store/model/` and then include it in `src/store/model/index.ts` like the `acompc` and `zotac` ones, it'll work perfectly.

**Reply by Pluegpu** [reply]
Worked like a charm. I didnt see you had to add it twice in the indexfile. My bad. Thanks a lot!!

**Reply by Aragorn450** [reply]
Great, glad to hear it! Please do mark it as the answer then. That way others know this issue is answered too.
